dataplane/internal/kernel/kerneltest/kernel.go

```diff
@@ -0,0 +1,223 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// package kerneltest contains fake implemetation of structs in kernel package
+package kerneltest
```

Comment:
This package seems to be named too generically. An outside package would not expect "kerneltest.New()" to return an InterfaceManager type.

Reply:
Makes sense